mono/mini/profiler.c

```diff
@@ -0,0 +1,233 @@
+/*
```

Comment:
Files need to be unique across the runtime as some targets can't handle this.
We had to do some renaming in the past because of this.